website/docs/foundations/tokens/index.js

```diff
@@ -26,18 +26,27 @@ export default class Index extends Component {
     super(...arguments);
     this.groupedTokens = {};
     // prepare the tokens grouped by category
-    TOKENS_RAW.forEach((token) => {
-      const category = token.attributes.category;
-      if (!this.groupedTokens[category]) {
-        this.groupedTokens[category] = [];
-      }
-      // add an extra "aliases" attribute if other tokens are alias of it
-      const aliases = getAliases(token, TOKENS_RAW);
-      if (aliases.length > 0) {
-        token.aliases = aliases;
-      }
-      this.groupedTokens[category].push(token);
-    });
+    // alphabetize the filtered icons by category and then by name
```

Comment:
All the other features are fine to replicate, but I don't think we should alphabetize the tokens list. I assumed this change would be to the way the search was working (so it stops displaying the headings when there are no results for a given category). I apologize for the assumption and for not clarifying before. 

Alphabetizing the tokens list results in some of the tokens being jumbled up and not grouped by usage, which was the initial intention. Can we retain that approach, please?

Reply:
+1 on non-alphabetizing the tokens

Reply:
Ok I'll undo that part

Reply:
Done. Pushed up.